apps/server/src/modules/learnroom/service/course-do.service.ts

```diff
@@ -45,12 +45,27 @@ export class CourseDoService implements AuthorizationLoaderServiceGeneric<Course
 		await this.courseRepo.save(course);
 	}
 
-	public async startSynchronization(course: Course, group: Group): Promise<void> {
+	public async startSynchronization(
+		course: Course,
+		group: Group,
+		students: GroupUser[],
+		teachers: GroupUser[]
+	): Promise<void> {
 		if (course.syncedWithGroup) {
 			throw new CourseAlreadySynchronizedLoggableException(course.id);
 		}
 
 		course.syncedWithGroup = group.id;
+		course.name = group.name;
```

Comment:
We already have this logic in the schulconnex-course-sync.service.ts. We should not have the same logic twice.

Also I don't think the name should be updated for an existing course.